examples/src/main/java/ai/djl/examples/inference/benchmark/util/Arguments.java

```diff
@@ -258,14 +258,14 @@ public String getOutputDir() {
     }
 
     public Class<?> getInputClass() {
-        if (inputShapes == null) {
+        if (inputShapes.isEmpty()) {
```

Comment:
how about inputShapes == null || inputShapes.isEmpty()

Reply:
inputShapes is PairList and initialized in constructor.